DEV.md

```diff
@@ -35,8 +35,10 @@ RUN go get github.com/microcosm-cc/bluemonday && \
 
 RUN CGO_ENABLED=0 GOOS=linux go build -a -installsuffix cgo -o app .
 
-ADD https://github.com/openfaas/faas/releases/download/0.8.0/fwatchdog /usr/bin
-RUN chmod +x /usr/bin/fwatchdog
+RUN apt-get update && apt-get install -y make curl \
```

Comment:
I think for the example it's OK to keep `ADD` for brevity and then show the `curl` method as an optimization underneath.

Reply:
I agree, I'll update it then.